UPGRADE-6.3.md

```diff
@@ -3,22 +3,20 @@ UPGRADE FROM 6.2 TO 6.3
 
 # Essentials
 
-## Changed Signatures
-
-See the Changed Interfaces and Method Signatures section whether changes in signatures affect the project.
-
-# Changed Features
+The steps in this chapter are required to get the project up and running in version 6.3.
 
 ## Symfony 3.4
 
 The system now uses Symfony 3.4, which needs a few adjustments:
 
-In dev mode the debug toolbar will report deprecations concerning services that are retrieved through the
-ServiceLocator or the Symfony container directly. To be prepared for Symfony 4, ServiceLocator and container calls
-should be used as rarely as possible (no news here) and dependency injection should be preferred. Where dependency
-injection is not possible, services should be declared public explicitly. The deprecation warnings will also be
-logged, potentially leading to huge log files and if there is a large number of warnings, performance in the dev
-environment will degrade. Therefore it is recommended to deal with most of the deprecations.
+In dev mode the debug toolbar will report some deprecations concerning non-public services that are retrieved through
+the ServiceLocator or the Symfony service container directly. In Symfony 4 this will no longer be possible; in addition,
+services will be private by default in Symfony 4.
```

Comment:
doubled Symfony 4

Reply:
Yes, but this redundancy makes clear that both changes are to be expected in Symfony 4. Would keep it as is.